docs/reference/aggregations/bucket/children-aggregation.asciidoc

```diff
@@ -11,7 +11,7 @@ For example, let's say we have an index of questions and answers. The answer typ
 
 [source,js]
 --------------------------------------------------
-PUT child_example?include_type_name=true
+PUT child_example
 {
   "mappings": {
     "_doc": {
```

Comment:
To update this example, we'll also need to remove the type name `_doc` from the mappings definition. I think this won't show up as a docs failure, because this example isn't actually run in tests.

Reply:
I see, I was under the false impression that the parameter was added solely to prevent failing docs-tests.
In that case probably large parts of this PR are void, but I think at least a huge chunk of the cases e.g. under "analysis" don't touch mappings at all, so I'll see if I can skim these quickly and maybe just restrict the PR to those simple cases.